hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml

```diff
@@ -67,38 +67,42 @@
             name="context"
         >{'search_default_previous_month': 1, 'search_default_current_month': 1, 'search_default_my': 1}</field>
         <field name="view_mode">pivot,graph</field>
+        <field
+            name="view_ids"
+            eval="[(5, 0, 0), (0, 0, {'view_mode': 'pivot', 'view_id': ref('hr_attendance_theoretical_view_pivot')}), (0, 0, {'view_mode': 'graph', 'view_id': ref('hr_attendance_theoretical_view_graph')})]"
+        />
     </record>
-    <record id="hr_attendance.menu_hr_attendance_report" model="ir.ui.menu">
+    <record id="hr_attendance.menu_hr_attendance_reporting" model="ir.ui.menu">
         <field
             name="groups_id"
-            eval="[(4, ref('hr_attendance.group_hr_attendance'))]"
+            eval="[(4, ref('hr_attendance.group_hr_attendance_officer'))]"
         />
         <!-- We need to leave the action empty for consistency because we are going
             to set different submenus, otherwise the Reports menu would not be
             displayed to a basic user. !-->
         <field name="action" eval="False" />
     </record>
     <menuitem
-        id="menu_hr_attendance_report"
+        id="menu_hr_attendance_reporting"
```

Comment:
I think this would not need to be changed.

Reply:
That has nothing to do with this one, as this is a new XML-ID not linked with the main one. Please respect existing XML-IDs.